app/config/api.py

```diff
@@ -58,6 +59,19 @@ def exception_to_response(request: HttpRequest, exception: Exception) -> HttpRes
     )
 
 
+@api.exception_handler(EndpointConnectionError)
+def handle_cognito_connection_error(
+    request: HttpRequest, exception: EndpointConnectionError
+) -> HttpResponse:
+    return api.create_response(
+        request,
+        {
+            "code": 503, "description": "Service Unavailable"
```

Comment:
Same as in #34 : Might want to [omit "code" and rename "description" to "detail"](https://github.com/geoadmin/service-control/pull/34#discussion_r1822801880).

Reply:
See #35